python/shark_turbine/dynamo/importer.py

```diff
@@ -581,6 +581,12 @@ def _import_torch_op_overload(
             isinstance(node.args[1], float) or isinstance(node.args[1], int)
         ):
             mlir_op_name = TENSOR_SCALAR_OP_CONVERTER[mlir_op_name]
+            # replace schema to retrieve the right function signature
```

Comment:
Can you leave an example or two of the op names we are massaging here in the comment. Will help those who follow know what string munging this is doing.

Reply:
Sure, this particular snippet isn't addressing ops outside of `TENSOR_SCALAR_OP_CONVERTER` but rather is making sure we get the right schema information - previously this was fine as we simply didn't use the `expected_jit_type` argument for converting scalars but now we do - so an incorrect jit type will cause an issue.